automated install/basic-install.sh

```diff
@@ -57,6 +57,18 @@ Cloudflare (DNSSEC);1.1.1.1;1.0.0.1;2606:4700:4700::1111;2606:4700:4700::1001
 EOM
 )
 
+DNS_SERVERSv6ONLY=$(
+    cat <<EOM
+Google (ECS, DNSSEC);2001:4860:4860:0:0:0:0:8888;2001:4860:4860:0:0:0:0:8844
+OpenDNS (ECS, DNSSEC);2620:119:35::35;2620:119:53::53
+DNS.WATCH (DNSSEC);2001:1608:10:25:0:0:1c04:b12f;2001:1608:10:25:0:0:9249:d69b
```

Comment:
We should not offer DNS.WATCH anymore.

https://github.com/pi-hole/FTL/pull/2411